src/Nav.php

```diff
@@ -184,6 +192,40 @@ public function currentPath(string $value): self
         return $new;
     }
 
+    /**
+     * Whether to fade the nav items when toggling between them.
+     *
+     * @param bool $value Whether to fade the nav items when toggling between them. Defaults to `true`.
+     *
+     * @return self A new instance with the specified fade value.
+     */
+    public function fade(bool $value): self
+    {
+        if ($this->isTabsOrPills() === false && $value) {
+            throw new RuntimeException('Fade effect can only be used with tabs or pills.');
+        }
+
+        $new = clone $this;
+        $new->fade = $value;
+
+        return $new;
+    }
+
+    /**
+     * Sets the ID of the nav component.
+     *
+     * @param bool|string $value The ID of the alert component. If `true`, an ID will be generated automatically.
+     *
+     * @return self A new instance with the specified ID.
+     */
+    public function id(bool|string $value): self
```

Comment:
```suggestion
    public function id(bool|string $id): self
```

Reply:
When it is a single argument, use value or values.